scaffolding-chef/lib/scaffolding.sh

```diff
@@ -57,7 +57,7 @@ do_default_build_service() {
 
 chef_client_cmd()
 {
-  chef-client -z -l {{cfg.log_level}} -c $pkg_svc_config_path/client-config.rb -j $pkg_svc_config_path/attributes.json --once --no-fork --run-lock-timeout {{cfg.run_lock_timeout}}
+  chef-client -z -l {{cfg.log_level}} -c $pkg_svc_config_path/client-config.rb -j $pkg_svc_config_path/attributes.json --once --no-fork --run-lock-timeout {{cfg.run_lock_timeout}} --chef-license={{cfg.chef_license.acceptance}}
```

Comment:
1) The `--chef-license` flag will cause an error on `chef-client` versions < 15, so we recommend either setting the `CHEF_LICENSE` environment variable or adding `chef_license "accept-no-persist` to the `client-config.rb` instead. Those are both backwards compatible. Is it possible to do that via this scaffolding? If not, there should be some kind of version check.
2) If passing the flag I would recommend `--chef-license "{{cfg.chef_license.acceptance}}"`